src/promptflow-core/pyproject.toml

```diff
@@ -49,8 +49,8 @@ jsonschema = ">=4.0.0,<5.0.0"  # used to validate tool
 filetype = ">=1.2.0"  # used to detect the mime type for multi-media input
 flask = ">=2.2.3,<4.0.0"  # Serving endpoint requirements
 python-dateutil = ">=2.1.0,<3.0.0"  # Contracts RunInfo
-# ========executor-service dependencies========
-fastapi = ">=0.109.0,<1.0.0" # used to build web executor server
+# ========fastapi server dependencies========
+fastapi = ">=0.109.0,<1.0.0" # used to build fastapi server
```

Comment:
shall this keep in extra?

Reply:
Also, promptflow-devkit project also need this as optional includes.

Reply:
it's included in core already, no need to put in extra any more

Reply:
we keep the extra "executor-service", just leave it empty.

Reply:
We can keep this for now. But need to resolve in future.

Reply:
make sense to me, let's have a separate discussion about how to better manage serving & core dependencies.